packages/directory-plugin/README.md

```diff
@@ -0,0 +1,30 @@
+# @wmr/directory-plugin
+
+Allows you to import multiple files like this:
+
+```js
+import files from 'ls:./pages';
```

Comment:
`ls:` -> `directory:`

Reply:
a thought: what about `dir:`?

Reply:
I'm good with `dir` too, although the more verbose variant might be easier to for newcomers. Don't mind going either way though.